.github/workflows/osx_testing.yml

```diff
@@ -178,7 +178,7 @@ jobs:
       - name: Install testing dependencies
         run: |
           cd "${SRCDIR}"
-          pip3 install -r test-run/requirements.txt
+          pip3 install PyYAML gevent==22.10.2
```

Comment:
Shouldn't we bump it in test-run instead/as well?

Reply:
gevent-21.* is not compatible with python-3.11 (installed on macos).
I'll double-check the relevance of these edits.

Reply:
Yep (you said this in the commit message). But whether gevent-22.* incompatible with python <= 2.10?

Reply:
From gevent changelog:
'Support for legacy versions of Python, including 2.7 and 3.6, will be ending soon'

Reply:
Okay, the reason is compatibility with python < 2.11.

It is interesting how it works for Mac OS 11/12 for other projects, at least Tarantool itself. But it is not a question for this patch, so I'm closing the discussion.